nebula_ros/launch/continental_launch_all_hw.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<launch>
+    <arg name="sensor_model" default="ARS548"/>
+    <arg name="frame_id"  default="continental_ars548"/>
+
+    <arg name="sensor_ip" default="10.13.1.113" description="Radar Sensor IP"/>
+    <arg name="host_ip" default="10.13.1.166" description="Broadcast IP from Sensor"/>
+    <arg name="multicast_ip" default="224.0.2.2" description="Multicast Sensor IP"/>
+    <arg name="data_port" default="42102" description="LiDAR Data Port"/>
+    <arg name="configuration_host_port" default="42401" description="LiDAR GNSS Port"/>
+    <arg name="configuration_sensor_port" default="42101" description="LiDAR GNSS Port"/>
+
+    <arg name="setup_sensor" default="False" description="Enable sensor setup on hw-driver."/>
```

Comment:
This argument is unused. There does not seem to be any setup communication for this sensor, so this can be removed.

Reply:
:bow: 
Addressed in 
182c571